user/views.py

```diff
@@ -43,7 +43,7 @@ def privacy():
         user = User.objects.get(id=current_user.id)
         return render_template('user/privacy.html',
                            user=user,
-                           locations=user.locations)
+                           locations=reversed(user.locations))
```

Comment:
Where does thie `reversed` function come from?

Reply:
Does this actually work?

Reply:
This actually works ! After some research, I've found that the difference between `reversed` and `list.reverse` is that `reversed` returns the reversed list but leaves the original list unchanged while `list.reverse` reverses the list but does not return the result. To user `list.reverse`, I have to add the line `user.locations.reverse()` before the `return`. What do you prefer ?

More information about `reversed` : https://docs.python.org/fr/3/library/functions.html?highlight=reversed#reversed

Reply:
That's indeed simple and great!